ee/dtr/user/manage-images/sign-images/index.md

```diff
@@ -7,53 +7,52 @@ redirect_from:
 - /ee/dtr/user/manage-images/sign-images/manage-trusted-repositories/
 ---
 
-2 Key components of the Docker Trusted Registry are the Notary Server and Notary
-Signer. These 2 containers give us the required components to use Docker Content
-Trust right out of the box. [Docker Content
-Trust](/engine/security/trust/content_trust/) allows us to sign image tags,
+Two key components of the Docker Trusted Registry are the Notary Server and the Notary
+Signer. These two containers offer the required components for using Docker Content
+Trust (DCT) out of the box. [Docker Content
+Trust](/engine/security/trust/content_trust/) allows you to sign image tags,
 therefore whoever pulls the image can validate that they are getting the image
-you create, or a forged one. 
+you created. 
 
-As part of Docker Trusted Registry both the Notary server and the Registry
-server are accessed through a front end Proxy, with both components sharing the
-UCP's RBAC Engine. Therefore no additional configuration of the Docker Client
-is required to use trust.
+As part of DTR, both the Notary and the Registry
+servers are accessed through a front-end proxy, with both components sharing the
+UCP's RBAC (Role-based Access Control) Engine. Therefore, you do not need additional Docker client
+configuration in order to use DCT.
 
-Docker Content Trust is integrated into the Docker CLI, allowing you to
-configure repositories, add signers and sign images all through the `$ docker
-trust` command.
+DCT is integrated with the Docker CLI, and allows you to:
+- Configure repositories
+- Add signers, and 
+- Sign images using the `docker trust` command
 
 ![image without signature](../../../images/sign-an-image-1.svg)
 
 ## Sign images that UCP can trust
 
 UCP has a feature which will prevent [untrusted
 images](/ee/ucp/admin/configure/run-only-the-images-you-trust/) from being
-deployed on the cluster. To use this feature, we first need to upload and sign
-images into DTR. To tie the signed images back to UCP, we will actually sign the
-images with private keys of UCP users. Inside of a UCP Client bundle the
-`key.pem` can be used a User's private key, with the `cert.pem` being a public
-key within a x509 certificate. 
+deployed on the cluster. To use the feature, you need to sign and push images to your DTR. 
+To tie the signed images back to UCP, you need to sign the
+images with the private keys of the UCP users. From a UCP client bundle, you can use
```

Comment:
Ahh this is my mistake to, but it needs to be more forceful, instead of "you can use" should be "you need to use the key.pem as the private key... etc." Otherwise the UCP feature won't work :S